kedro_vertexai/cli.py

```diff
@@ -101,16 +110,65 @@ def run_once(
     context_helper = ctx.obj["context_helper"]
     config: RunConfig = context_helper.config.run_config
     client: VertexAIPipelinesClient = context_helper.vertexai_client
+    image: str = image if image else config.image
+
+    reminder = not config.no_reminder
+    if auto_build:
+        try:
```

Comment:
I would refactor the whole logic related to auto_build into a separate function, you can put it in `utils.py`.

Reply:
done